### ISSUE TITLE
Support for braille input from the PC keyboard

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1003]
**Reported by RamboUdin on 2010-10-28 06:27**
  With braille display, you can control the computer using braille (as far as i know)[JAWS 12, i think](with).
Can NVDA support that feature as well? But, NVDA will use the keyboard to perform that. Maybe, it is 8 dots, etc.
(like the Duxbury braille translator)

THANKS.

**Blocked by #808**

**Comment by leonardder** [comment]
@jcsteh commented:
> I provided a global plugin to do this as part of my testing of #808. Whether this should be part of NVDA itself is yet to be determined, but my feeling right now is that it should be an add-on.

I assume your thoughts haven't changed yet? In that case, I guess this can be closed.

**Comment by jcsteh** [comment]
I think this is still best as an add-on. It's available as the [PC Keyboard Braille Input](https://addons.nvda-project.org/addons/pcKeyboardBrailleInput.en.html) add-on.
